schema.sql

```diff
@@ -10,3 +10,31 @@ weight_kg DEC NOT NULL
 );
 
 ALTER TABLE animals ADD species TEXT;
+
+CREATE TABLE owners(
+id INT GENERATED ALWAYS AS IDENTITY,
+full_name TEXT NOT NULL,
+age INT,
+PRIMARY KEY(id)
+);
```

Comment:
- [ ] [REQUIRED] Good job so far! On line 16 should like this `full_name TEXT NOT NUL varchar(255),` Please fix this.